packages/core/src/card/card.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components(Preview)/Card/Proposal" />
+
+# Card proposal
+
+<cds-alert-group status="info" cds-layout="m-b:lg">
+  <cds-alert>Progress bars are still missing in `core`.</cds-alert>
+</cds-alert-group>
+
+## Basic Usage
+
+```typescript
+import '@cds/core/card/register.js';
+```
+
+```html
+<cds-card>
+  <cds-progress-bar></cds-progress-bar>
+
+  <header>{ header }</header>
+
+  <cds-divider></cds-divider>
+
+  <div>{ main content }</div>
+
+  <cds-divider></cds-divider>
+
+  <cds-progress-bar></cds-progress-bar>
+
+  <footer>{ footer }</footer>
+</cds-card>
```

Comment:
Is this file supposed to be in the PR? This looks like card component specific not panel

Reply:
This is just the API proposal of the card. If you think it is better to be added as a separate PR, let me know.

Reply:
Yeah, we should keep it separate. Storybook is still the primary docs for Core so this would show up on the site with the next release.